docs/configuration/configuration_installation.rst

```diff
@@ -24,31 +24,30 @@ The application should have the access attributes (e.g, URL, username, password,
 * When the app runs for the first time, it will look for the application configuration file.
 
   * If the file does not exist, it will copy a default file to the user's home folder.
-  * The file contains some default values which the user can modify.
+  * The file contains some default values that the user should modify.
 
 * The user needs to install Elasticsearch_ before start using the app.
 
-* The user needs to create the Elasticsearch_ indices and insert the required data in order to use the application.
+* The user may need to create the Elasticsearch_ indices and insert the required data in order to use the application.
 
 * The method ``get_index_data_from_database`` inside :omero_search_engine:`manage.py <manage.py>` allows indexing automatically from the app.
 
-* Another method to index the data:
+* Index the data using CSV files:
 
-  * The data is extracted from the IDR/OMERO database using some SQL queries and saved to csv files using :omero_search_engine:`sql_to_csv.py <omero_search_engine/cache_functions/elasticsearch/sql_to_csv.py>`
+  * The data should be extracted from the IDR/OMERO database using some SQL queries and saved to CSV files using :omero_search_engine:`sql_to_csv.py <omero_search_engine/cache_functions/elasticsearch/sql_to_csv.py>`
   * The image index data is generated in a large file, so it is recommended that the user splits it into several files to facilitate the processing of the data and its insertion into the index e.g. ``split -l 2600000 images.csv``.
-  * ``create_index``: Create the Elasticsearch indices, it can be used to create a single index or all the indices; the default is creating all the indices.
-  * the indices are saved using the script :omero_search_engine:`sql_to_csv.py <omero_search_engine/cache_functions/elasticsearch/elasticsearch_templates.py>`
-  * ``add_resource_data_to_es_index``: Insert the data to the ELasticsearch index; the data can be in a single file (CSV format) or multiple files.
-
+  * ``create_index``: Create the Elasticsearch indices no data have been previously added, it can be used to create a single index or all the indices; the default is creating all the indices.
```

Comment:
```suggestion
  * ``create_index``: Create the Elasticsearch indices no data have been previously added, it can be used to create a single index or all the indices; the default is to create all the indices.
```